src/CoreBundle/Resources/public/javascript/table.js

```diff
@@ -181,52 +183,150 @@ $(document).ready(function () {
                 newOption: true
             }
         }
-    }).on('select2:select', function (e) {
-        if (e.params.data.newOption) {
-            var listname = searchLookup.data('listname');
-            document[listname]._startRecord.value = 0;
-            document[listname].submit();
+    }).on('select2:clear', function(e) {
+        const form = createSearchWordInputWithValue();
+
+        uiEnabled = false;
```

Comment:
uiEnabled (only) means "clear requested"?

Reply:
Yes, kind of (as long as blocking th ui isn't necessary aywhere else). Do you think it better to change the variable name?

Reply:
Yes, please.

Reply:
Done.